index.html

```diff
@@ -85,7 +86,7 @@ <h2>Canopy</h2>
           <li><button class="custom-buttonds" type="button">css</button></li>
           <li><button class="custom-buttonds" type="button">JavaScript</button></li>
         </ul>
-        <button class="big-buttonds" type="button">See Project</button>
+        <button id="buttonds1" class="big-buttonds" type="button">See Project</button>
       </div>
```

Comment:
- [ ] <b>REQUIRED:</b> The projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.**

Reply:
I can't really understand what is the issue here, can you please explain more.

Reply:
> * [ ]  REQUIRED: The projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.**

- [ ] What the CR means, is that project section, in order to associate each project with the popup window details, you must use a JavaScript array to store all of the information for all projects. So, you should remove the HTML code and store it in a javascript object. Kindly read more on [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) second to last point explains in details